packages/react-core/src/helpers/OUIA/OUIA.md

```diff
@@ -0,0 +1,93 @@
+---
+id: Open UI Automation
+section: developer-resources
+---
+
+## About Open UI Automation
+
+Open UI Automation (OUIA) is a specification of certain key guidelines to follow when creating a new
+web framework or application. The goal is to ease the burden of individuals wishing to create and
+maintain automated testing environments. OUIA standardizes some HTML attributes and web page
+behavior. It helps to create trivial and unambiguous locators of the elements in the DOM for
+various automated test suites.
+
+### Why not to use `id`, `ARIA` etc.?
+
+`ARIA` is designed as an accessibility specification and it doesn't account aspects that are crucial
+for automation such as a component type. `id` attribute might be reserved by some frontend code and
+there might be restrictions of changing it. Having a separate id attribute makes it application and
+framework independent. Developers and testers have a contract that these attributes are designed for
+UI automation.
+
+### Specification parts
+
+The following blocks exist in the specification:
+
+* `OUIA:Component` - components and component frameworks;
+* `OUIA:Page` - page identification;
+* `OUIA:PageSafe` - interaction safety.
+
+The complete description you can find in [https://ouia.readthedocs.io](https://ouia.readthedocs.io).
+
+[Selected](#list-of-ouia-compliant-components) PatternFly 4 components are `OUIA:Component`
+compliant.
+
+### Usage
+
+PatternFly 4 generates `data-ouia-component-type` attribute for selected components automatically
```

Comment:
```suggestion
PatternFly 4 generates a `data-ouia-component-type` attribute for selected components automatically
``` 
Adding the "a" made this sound clearer, but I'm not sure if that changes the technical meaning at all. Feel free to apply as you see fit!